hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/KafkaPushJob.groovy

```diff
@@ -179,4 +185,26 @@ class KafkaPushJob extends HadoopJavaJob {
     this.disableAuditing = disableAuditing;
     setJobProperty("disable.auditing", disableAuditing);
   }
+
+  /**
+   * DSL usesDisableAuditing method causes multi.topic=value to be set in the job file.
+   *
+   * @param multiTopic
+   */
+  @HadoopDslMethod
+  void usesMultiTopic(Boolean multiTopic) {
+    this.multiTopic = multiTopic;
+    setJobProperty("multi.topic", multiTopic);
```

Comment:
If the property is not set, don't you want to set the default values on the DSL side? If not, can you confirm if the null checks are happening on the Azkaban Job side?

Reply:
There is a default value set in the Azkaban job itself to set this to "False" such that it's backwards compatible. Is the convention to set a default value? The other properties added for this specific DSL haven't added any defaults in the DSL, only in the job.

Reply:
If you can update the PR's javadocs accordingly, I'm good.